keyboards/rephlex/2k/2k.c

```diff
@@ -0,0 +1,154 @@
+/* Copyright 2023 RephlexZero (@RephlexZero)
+SPDX-License-Identifier: GPL-2.0-or-later */
+#include <stdint.h>
+#include "2k.h"
+#include "quantum.h"
+#include "analog.h"
+#include "eeprom.h"
+#include "analogkeys.h"
+
+extern pin_t matrix_pins[MATRIX_ROWS][MATRIX_COLS];
+void         bootmagic_lite(void) {
+    if (analogReadPin(matrix_pins[BOOTMAGIC_LITE_ROW][BOOTMAGIC_LITE_COLUMN]) < 1350) {
+        bootloader_jump();
+    }
+}
+
+#ifdef DEBUG_ENABLE
+static uint8_t i = 0;
+void           housekeeping_task_user(void) {
```

Comment:
```suggestion
void           housekeeping_task_kb(void) {
```

(and user doesn't need to be called here)